runtime-config.md

```diff
@@ -1,3 +1,8 @@
+# Host-specific container configuration
+
+The bundle's top-level directory MUST contain a configuration file called `runtime.json` with [UTF-8][] [JSON][].
+For now the canonical schema is defined in [`runtime_config.go`](runtime_config.go) and [runtime_config_linux.go](runtime_config.go), but this will be moved to a formal JSON schema over time.
```

Comment:
On Fri, Sep 04, 2015 at 11:44:48AM -0700, Vincent Batts wrote:
> and "moved to a formal JSON schema over time" where and why this comment?

Both are just echoing the current host-independent phrasing [1].  I
tried to make that clear in the commit message [2] and original PR
post with the:

  “To get a place to say this for the runtime.json docs, I used
  config.md as a template for the top-level header and blurb.”

text, but I'm happy to use different commit-message phrasing if that
would help make the source more obvious.

[1]: https://github.com/opencontainers/specs/blob/9d59c53bbf117075424f8bbf82b605a5c59b53f7/config.md#configuration-file